### PR TITLE
Changes to be compliant with ENA's Portal API v2.0

### PR DESCRIPTION
**Pull request by mberacochea**
The biggest change is related to the samples.

Upgraded the ena-api-handler, I've also included it the base reqs.txt file (pinning the version).

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -35,7 +35,6 @@ jobs:
 
     - name: 🔧 - Install Dependencies
       run: |
-        pip install -U git+https://github.com/EBI-Metagenomics/ena-api-handler.git
```
Comment (SandyRogers) [inline .github/workflows/test.yml:38]: Nice! Also need to remove this line from the repo README and from the Dockerfiles in `docker/`.

**Review by SandyRogers — COMMENTED** [review]
Nice! Just need to remove the separate installation of the ena-api-handler from the dockerfiles (which are used for running the API in the mgnify-web docker-compose setup) and update the repo readme.

File: .github/workflows/test.yml (hunk 1 above)
Comment (mberacochea) [reply .github/workflows/test.yml:38]: I'll do it in a bit then :)

File: requirements.txt [hunk 2]
```diff
@@ -9,7 +9,7 @@ mysqlclient==2.1.1
 mysql-connector-python~=8.0.23
 sqlparse==0.4.3
 whitenoise==6.4.0
-requests==2.28.2
+requests==2.28.1
```
[the remark this answers is not in the document]
Comment (mberacochea) [reply requirements.txt:12]: ena-api-handler requires 2.28.1, I'll upgrade both later.

**Pull request merged**